rmw_opendds_cpp/src/type_support_common.hpp

```diff
@@ -55,6 +55,23 @@
     } \
   }
 
+inline const rosidl_message_type_support_t *
+rmw_get_message_type_support(const rosidl_message_type_support_t * type_support)
+{
+  if (!type_support) {
+    RMW_SET_ERROR_MSG("type support is null");
+    return nullptr;
+  }
+  const rosidl_message_type_support_t * ts =
```

Comment:
Since the OpenDDS implementation of ROSIDL TypeSupport is not really ready for use, consider an interim implementation of this method (`rmw_get_message_type_support`) that returns a "dummy" version of `rosidl_message_type_support_t`

Reply:
@mitza-oci Does the most recent change work for you?

Reply:
I guess so?  Not really in scope here, so I'll mark it resolved.